spec/models/user_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
## Integration test Passed 🥇 

![Capture](https://user-images.githubusercontent.com/63972670/190844742-f6bbf161-ef1b-4250-9d9b-f0acee7a5125.JPG)

Reply:
thank you @OLIPLICHE for the constructive feedback